quinn-proto/src/connection/mod.rs

```diff
@@ -536,6 +546,12 @@ where
             }
         }
 
+        if buf.is_empty() && !was_congestion_blocked {
+            self.app_limited = true;
+        } else {
+            self.app_limited = false;
+        }
```

Comment:
Let's just write this as `self.app_limited = buf.is_empty() && !was_congestion_blocked;`?

Reply:
sure. Fixed